src/App.jsx

```diff
@@ -96,8 +96,8 @@ class App extends Component {
                 this.setState({ responded: { won } });
                 if (won) {
                   this.setState({ points: points + 1, timer: newTimer + 3000 });
-                } else {
-                  this.setState({ timer: newTimer - 1000 });
+                } else if (points > 0) {
+                  this.setState({ points: points - 1 });
```

Comment:
Neat!

Note however how this unintentionally makes failed attempts reset the timer. Since the time left is passed through `newTimer` we need to update our timer state to reflect this.

Either this is the expected behavior (although a bit strange since a correct answer will decrease the timer), or it would be good to open an issue on this bug.